packages/relay/src/lib/relay.ts

```diff
@@ -123,8 +123,10 @@ export class RelayImpl implements Relay {
     }
 
     client.setTransportSecurity(process.env.CLIENT_TRANSPORT_SECURITY === 'true' || false);
+    client.setRequestTimeout(parseInt(process.env.SDK_REQUEST_TIMEOUT || ''));
```

Comment:
Add this new env var to env.example, readme, localAcceptance, helm-chart. Also if this is not set it's going to return NaN, because `parseInt('')` is NaN.

Reply:
Updated.

Reply:
I still see `parseInt('')` did you push?

Reply:
Should it be?
```suggestion
    client.setRequestTimeout(parseInt(process.env.SDK_REQUEST_TIMEOUT || '10000'));
```